pkg/phlaredb/compact.go

```diff
@@ -417,9 +417,6 @@ func compactMetas(src ...block.Meta) block.Meta {
 			labels[k] = v
 		}
 	}
-	if hostname, err := os.Hostname(); err == nil {
-		labels[block.HostnameLabel] = hostname
-	}
 	meta.Source = block.CompactorSource
 	meta.Compaction = tsdb.BlockMetaCompaction{
 		Deletable: false,
```

Comment:
Again wrong line, look more 404/405. The MaxTime and MinTime is not consistent with the one of the block itself. 

While we use integer milliseconds for parents, we use a floating point number for MaxTime, MinTime for the block itself. We should fix that:

```
{
        "ulid": "01HCW5635RGVR5DV2HEQJNPXCQ",
        "minTime": 1697316520.195,
        "maxTime": 1697317199.969,
[...]
        "compaction": {
                "parents": [
                        {
                                "ulid": "01HCQZ5HA3DHKSJPK5EGJ0YKDK",
                                "minTime": 1697316520195,
                                "maxTime": 1697316599969
                        },
[...]
```

Reply:
Made the change, I had to fork the compaction meta struct from tsdb.